docs/howto/debugging.md

```diff
@@ -81,6 +82,23 @@ See also [in the "Troubleshooting" section below](#debug-remotely-never-loads).
 [chrome-devtools-device]: https://facebook.github.io/react-native/docs/debugging.html#debugging-on-a-device-with-chrome-developer-tools
 
 
+<div id="react-devtools">
+
+## React Devtools
+
+The standalone version of React DevTools, which is an electron app, can be used
+to debug  the react-native component hierarchy. Refer to
+[the docs](https://facebook.github.io/react-native/docs/debugging.html#react-sdeveloper-tools)
+for instructions on how to use with React Native.
+
+As of today's month, we do not support the latest version (v4) of
+react-devtools, and v3 needs to be installed.  This can be done using the
+command `yarn global add react-devtools@3`. See
+[this blog post](https://reactjs.org/blog/2019/08/15/new-react-devtools.html#which-versions-of-react-are-supported)
+for complete compability information.
+
+
+
```

Comment:
All lines are less than 80 columns now, so some lines may look tiny because the words following the newline are long.

Reply:
Doesn't look tiny to me!

In any case, yep, it's normal that some lines will be short when paragraphs are properly wrapped :slightly_smiling_face: